modules/include/concurrentqueue.h

```diff
@@ -0,0 +1,3747 @@
+// Provides a C++11 implementation of a multi-producer, multi-consumer lock-free queue.
```

Comment:
Don't we have this one somewhere else in the code?
https://github.com/klepsydra-technologies/concurrentqueue/tree/master

Reply:
Done